migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc

```diff
@@ -0,0 +1,47 @@
+[id='installing-restricted-3-4']
+= Installing the Migration Toolkit for Containers in a restricted network environment
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-3-4
+:installing-restricted-3-4:
+
+toc::[]
+
+You can install the {mtc-full} ({mtc-short}) on {product-title} 3 and on {product-title} {product-version} in a restricted network environment.
+
+[IMPORTANT]
+====
+You must install the same {mtc-short} version on all clusters.
+====
+
+By default, the {mtc-short} web console and the `Migration Controller` pod run on the target cluster.
+
+You can configure the `Migration Controller` custom resource manifest to run the {mtc-short} web console and the `Migration Controller` pod on a link:https://access.redhat.com/articles/5064151[source cluster or on a remote cluster].
+
+== Creating an Operator catalog in a local registry
+
+You must create a mirror image of the Operator catalog, push it to a local registry, and then create a catalog from the image.
+
+For details, review the following documentation:
```

Comment:
@bergerhoffer I think this sentence sounds clunky but I couldn't think of a better way to phrase it. Open to suggestions!

Reply:
I think I found a solution. I changed the whole mess of bullets to this:

You must create an Operator catalog from a mirror image in a local registry. See xref:../operators/admin/olm-restricted-networks.adoc[Using Operator Lifecycle Manager on restricted networks] for instructions.